src/components/Markdown/index.jsx

```diff
@@ -17,169 +17,177 @@ markdown.use(linkAttributes, {
+      '& table': {
+        width: '100%',
+        display: 'block',
+        overflowX: 'auto',
+        borderCollapse: 'collapse',
+        borderSpacing: 0,
+        overflow: 'hidden',
+        '& .prop-name': {
+          fontSize: 13,
+          fontFamily: 'Consolas, "Liberation Mono", Menlo, monospace',
+        },
+        '& .required': {
+          color: theme.palette.type === 'light' ? '#006500' : '#9bc89b',
+        },
+        '& .prop-type': {
+          fontSize: 13,
+          fontFamily: 'Consolas, "Liberation Mono", Menlo, monospace',
+          color: theme.palette.type === 'light' ? '#932981' : '#dbb0d0',
+        },
+        '& .prop-default': {
+          fontSize: 13,
+          fontFamily: 'Consolas, "Liberation Mono", Menlo, monospace',
+          borderBottom: `1px dotted ${theme.palette.text.hint}`,
+        },
       },
-      '& .required': {
-        color: theme.palette.type === 'light' ? '#006500' : '#9bc89b',
+      '& thead': {
+        fontSize: 14,
+        fontWeight: theme.typography.fontWeightMedium,
+        color: theme.palette.text.secondary,
       },
-      '& .prop-type': {
-        fontSize: 13,
-        fontFamily: 'Consolas, "Liberation Mono", Menlo, monospace',
-        color: theme.palette.type === 'light' ? '#932981' : '#dbb0d0',
+      '& tbody': {
+        fontSize: 14,
+        lineHeight: 1.5,
+        color: theme.palette.text.primary,
+      },
+      '& td': {
+        borderBottom: `1px solid ${theme.palette.divider}`,
+        padding: `${theme.spacing.unit}px ${2 * theme.spacing.unit}px ${
+          theme.spacing.unit
+        }px ${theme.spacing.unit}px`,
+        textAlign: 'left',
+      },
+      '& td:last-child': {
+        paddingRight: 3 * theme.spacing.unit,
+      },
+      '& td compact': {
+        paddingRight: 3 * theme.spacing.unit,
       },
-      '& .prop-default': {
+      '& td code': {
         fontSize: 13,
-        fontFamily: 'Consolas, "Liberation Mono", Menlo, monospace',
-        borderBottom: `1px dotted ${theme.palette.text.hint}`,
+        lineHeight: 1.6,
+      },
+      '& th': {
+        whiteSpace: 'pre',
+        borderBottom: `1px solid ${theme.palette.divider}`,
+        fontWeight: theme.typography.fontWeightMedium,
+        padding: `0 ${theme.spacing.unit * 2}px 0 ${theme.spacing.unit}px`,
+        textAlign: 'left',
+      },
+      '& th:last-child': {
+        paddingRight: 3 * theme.spacing.unit,
+      },
+      '& tr': {
+        height: 48,
+      },
+      '& thead tr': {
+        height: 64,
+      },
+      '& strong': {
+        fontWeight: theme.typography.fontWeightMedium,
+      },
+      '& blockquote': {
+        borderLeft: `5px solid ${theme.palette.text.hint}`,
+        padding: `${theme.spacing.unit / 2}px ${3 * theme.spacing.unit}px`,
+        margin: `${3 * theme.spacing.unit}px 0`,
+      },
+      '& a, & a code': {
+        // Style taken from the Link component
+        color: aColor,
```

Comment:
I think the preference is to put the ternary conditional here, rather than declaring a constant.  You could write it as `theme.palette.common[theme.palette.type === 'light' ? 'black' : 'white']` to make it a bit shorter.

Reply:
Okay, I'll update the changes!